netcdf_output.py

```diff
@@ -1105,25 +1105,14 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
 
         M11 = offset2vy_2 / (offset2vx_1 * offset2vy_2 - offset2vx_2 * offset2vy_1) / scale_factor_1
 
-        x1 = np.nanmin(M11[:])
-        x2 = np.nanmax(M11[:])
-        y1 = -50
-        y2 = 50
-
-        C = [(y2-y1)/(x2-x1), y1-x1*(y2-y1)/(x2-x1)]
-        # M11 = C[0]*M11+C[1]
-        var.setncattr('scale_factor', np.float32(1/C[0]))
-        var.setncattr('add_offset', np.float32(-C[1]/C[0]))
-
-        M11[noDataMask] = NoDataValue * np.float32(1/C[0]) + np.float32(-C[1]/C[0])
-        # M11[noDataMask] = NoDataValue
+        M11[noDataMask] = NoDataValue
```

Comment:
@mliukis for float32, wouldn't it make more sense to use `np.nan` for the NodataValue?

In the rest of this script, we use either `None` or `np.nan` for float32/64 variables:
![image](https://github.com/user-attachments/assets/609a3df5-50c9-4188-abe8-b47749e31e20)

Reply:
@jhkennedy hmm... We use _FillValue: -32767 for floating point values in datacubes, composites. May be a question for @alex-s-gardner?

Reply:
@jhkennedy Actually, when we say **fill_value = None**, it means that **_FillValue** is not applicable, like for **x** and **y**

Reply:
yes, None doesn't really matter here as it's saying there isn't missing/no data; just showing what all we do for floats. 

> Whatever NoDataValue we set, it gets converted to np.nan on read

Right, which is why I'm suggesting using np.nan.